ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java

```diff
@@ -36,9 +32,8 @@ public void setup() {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void XSSOnPhoneNumberFieldTest() throws Exception {
+    public void xssOnPhoneNumberFieldTest() throws Exception {
```

Comment:
What do you think of `xSSOnPhoneNumberFieldTest`  ?

Reply:
> `xSSOnPhoneNumberFieldTest` ?
I think this kind of convention doesnot in line with openmrs convention and probably java convention.

Reply:
AFAICT OpenMRS convention concerning test methods for selenium legacy tests uses class name just one thing that differs is that the class name begins with `UpperCase` and for test method begins with `LowerCase`. This aligns with Java method naming convention. See reference to Java naming convention [here](https://www.theserverside.com/feature/Java-naming-conventions-explained).

Below are the sample
A test method name with class called  [ContactInfoTest](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f2e3df96f7ff815ee8249c4dce61ae460339f770/ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java#L25) is [contactInfoTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f2e3df96f7ff815ee8249c4dce61ae460339f770/ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java#L38)
A test method name with class called [AddUserTest](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f2e3df96f7ff815ee8249c4dce61ae460339f770/ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java#L27) is [addUserTest()](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f2e3df96f7ff815ee8249c4dce61ae460339f770/ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java#L31)

Its from the artifacts above I recommend a test method name with class called [XSSOnPhoneNumberTest
](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f2e3df96f7ff815ee8249c4dce61ae460339f770/ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java#L28) should be xSSOnPhoneNumberFielldTest()